### PR TITLE
bulk-lookup: add vintage and total years to table

### PR DESCRIPTION
**Pull request by srabraham**
For https://github.com/burningmantech/ranger-clubhouse-web/issues/1140

### CODE REVIEW

File: app/controllers/admin/bulk-lookup.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ const FOUND_CSV_COLUMNS = [
   {title: 'Status', key: 'status'},
   {title: 'Email', key: 'email'},
   {title: 'Last Worked', key: 'last_worked'},
+  {title: 'Vintage', key: 'vintage'},
```
Comment (flwyd) [inline app/controllers/admin/bulk-lookup.js:13]: If you add `yesno: true` the CSV export will get Y and N values.

**Review by flwyd — CHANGES_REQUESTED** [review]
I'm not sure why the presubmit github actions aren't running; I'll look into it.

File: app/controllers/admin/bulk-lookup.js (hunk 1 above)
Comment (srabraham) [reply app/controllers/admin/bulk-lookup.js:13]: Done, in https://github.com/burningmantech/ranger-clubhouse-web/pull/1466

**Comment by srabraham** [comment]
Moving this PR to https://github.com/burningmantech/ranger-clubhouse-web/pull/1466, which is a PR in this repo, rather than from a different forked one

**Pull request closed without merging**